tools/js-sdk-release-tools/src/common/changelog/modifyChangelogFileAndBumpVersion.ts

```diff
@@ -78,11 +79,21 @@ export async function makeChangesForReleasingTrack2(packageFolderPath: string, p
     if(packageVersion.includes("beta")){
         pacakgeVersionDetail +=`\nCompared with version ${comparedVersion}`
     }
+    
+    const modularSDKType = getModularSDKType(packageFolderPath);
```

Comment:
do we need to review the breaking change for data-plane in spec pr? will this change impact the breaking detection there?